README.md

```diff
@@ -124,6 +124,7 @@ Currently **officially** using Airflow:
 1. [8fit](https://8fit.com/) [[@nicor88](https://github.com/nicor88), [@frnzska](https://github.com/frnzska)]
 1. [90 Seconds](https://90seconds.tv/) [[@aaronmak](https://github.com/aaronmak)]
 1. [99](https://99taxis.com) [[@fbenevides](https://github.com/fbenevides), [@gustavoamigo](https://github.com/gustavoamigo) & [@mmmaia](https://github.com/mmmaia)]
+1. [Acorns Grow](https://www.acorns.com/) [[Acorns](https://github.com/Acornsgrow)]
```

Comment:
Can you do it in a separate PR? Otherwise, this change will be difficult to cherry-pick to 1.10.X.

Reply:
Removed from this PR